src/ert/job_runner/runner.py

```diff
@@ -13,6 +13,7 @@ def __init__(self, jobs_data):
             os.environ["DATA_ROOT"] = self._data_root
 
         self.simulation_id = jobs_data.get("run_id")
+        self.experiment_id = jobs_data.get("experiment_id")
```

Comment:
Please also remember to add the `experiment_id` to the `Init` message (to have it available when the `job` messages is created).